css/index.css

```diff
@@ -96,26 +144,30 @@ p {
 }
 
 .middle-content .boxes {
+    font-size: 1.6rem;
     display: flex;
     flex-wrap: wrap;
     justify-content: space-evenly;
+    padding-bottom: 10px;
 }
 
 .middle-content .boxes .box {
     width: 12.5%;
     height: 100px;
     background: black;
-    margin: 20px 2.5%;
+    margin: 20px 2.4% 20px 2.8%;
     color: white;
     display: flex;
     align-items: center;
     justify-content: center;
+    text-align: center;
 }
 
 .bottom-content {
     display: flex;
-    margin: 0 2% 20px;
+    margin: -4px 1% 25px 4.3%;
```

Comment:
🤔

Reply:
Pixel perfection, haha... or tried it at least.